### PR TITLE
Fix code base on review comments

### PR DESCRIPTION
**Pull request by ifindthanh**
- Rollback ClientTest.java as default one
- Using sslcom-dv-ecc for SslComAcmeProvider instead of sslcom-dv-rsa
- Adding separated client test class for Ssl with EAB supported

### CODE REVIEW

File: acme4j-example/src/main/java/org/shredzone/acme4j/example/SSLClientWithEabTest.java [hunk 1]
```diff
@@ -0,0 +1,456 @@
...
+     *         Email address of account that owns the key information
+     */
+    public void fetchCertificate(Collection<String> domains, String eabKid, String eabHmacKey, String emailAddress) throws IOException, AcmeException {
+        // Load the user key file. If there is no key file, create a new one.
+        KeyPair userKeyPair = loadOrCreateUserKeyPair();
+
+        // Create a session for Let's Encrypt.
+        // Use "acme://letsencrypt.org" for production server
+        Session session = new Session("acme://ssl.com/staging");
+        //Session session = new Session("acme://letsencrypt.org/staging");
```
Comment (fergadis) [inline acme4j-example/src/main/java/org/shredzone/acme4j/example/SSLClientWithEabTest.java]:
```suggestion
        // Create a session for SSL.com.
        // Use "acme://ssl.com" for production server
        Session session = new Session("acme://ssl.com/staging");
```

File: ssl-truststore
Comment (fergadis) [inline ssl-truststore:1]: Please remove this file.

**Review by fergadis — CHANGES_REQUESTED** [review]

File: ssl-truststore
Comment (ifindthanh) [reply ssl-truststore:1]: it is already in my commit
image

**Review by codelux — APPROVED** [review]
changes accepted

**Pull request merged**